packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -114,9 +114,24 @@ class PictureLayer extends Layer {
   /// The picture's coodinate system matches this layer's coodinate system
   ui.Picture picture;
 
+  /// Hints that the painting in this layer is complex and would benefit from
+  /// caching.
+  ///
+  /// If this hint is not set, the compositor will apply its own huerstics to
+  /// decide whether the this layer is complex enough to benefit from caching.
+  bool isComplex = false;
```

Comment:
Changed to isComplexHint.  The problem with imperatives like "cache content" is that the implementation might not actually obey the imperatives.  It's better to use descriptions like "is complex" because that way the implementation can decide what to do with all the descriptions.